analyses/cell-type-wilms-tumor-06/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# pull base image
+FROM bioconductor/tidyverse:3.19
+
+
+# Change the default CRAN mirror
+RUN echo "options(repos = c(CRAN = 'https://mran.microsoft.com/snapshot/2022-02-01'), download.file.method = 'libcurl')" >> ${R_HOME}/etc/Rprofile.site
```

Comment:
> From experience ([AlexsLemonade/OpenPBTA-analysis#1641](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1641) 😅 ), I know why you have to set an MRAN snapshot from some time ago, but I'm not sure how nicely it will "play" with the Bioconductor base image using `3.19`.
> 
> If you are concerned about being tied to specific versions, I think one way to approach this would be to use the [`renv` package](https://rstudio.github.io/renv/articles/renv.html) in conjunction with Docker.
> 
> Here are some links to the OpenScPCA docs that you might find helpful:
> 
> * https://openscpca.readthedocs.io/en/latest/ensuring-repro/managing-software/using-renv/
> * https://openscpca.readthedocs.io/en/latest/ensuring-repro/docker/docker-images/#r-based-images
> 
> We can file an issue to add `renv` later, though; it doesn't need to happen in this pull request!

Thank you for pointing to these links! I'll try to build the docker as you suggested, sorry I missed this before!
![image](https://github.com/user-attachments/assets/567270a7-96fd-4369-8ef5-b336c6acd539)